src/core/parameters.jl

```diff
@@ -177,22 +177,13 @@ function get_column_names(key::ParameterKey, c::ParameterContainer)
     return get_column_names(key, get_multiplier_array(c))
 end
 
-function _set_parameter!(
-    array::AbstractArray{Float64},
-    ::JuMP.Model,
-    value::Float64,
-    ixs::Tuple,
-)
-    array[ixs...] = value
-    return
-end
-
+const _ValidDataParamEltypes = Union{Float64, IS.FunctionData, Tuple{Vararg{Float64}}}
```

Comment:
Any reason for this to be here and with an `_`?

Reply:
Not really, was just trying to communicate the private-ness of it. But I'll remove the underscore.